doc/install.md

```diff
@@ -0,0 +1,288 @@
+(install)=
+# Installation Guide
+
+If you have any questions or problems while installing `hnn_core`, feel free to
```

Comment:
do you think this should be a github issue instead of a discussion? In my head discussions are better used for conceptual questions